docs/commerce/checkout/configure_checkout.md

```diff
@@ -0,0 +1,155 @@
+                    metadata:
+                        next_step: summary
+                        controller: Ibexa\Bundle\Checkout\Controller\CheckoutStep\ShippingStepController::renderStepView
+                        label: 'Shipping & payment method'
+                        translation_domain: checkout
+                summary:
+                    from:
+                        - shipping_selected
+                        - summarized
+                    to: summarized
+                    metadata:
+                        next_step: complete
+                        controller: Ibexa\Bundle\Checkout\Controller\CheckoutStep\SummaryStepController::renderStepView
+                        label: 'Payment & summary'
+                        translation_domain: checkout
+                complete:
+                    from: summarized
+                    to: completed
+                    metadata:
+                        controller: Ibexa\Bundle\Checkout\Controller\CheckoutStep\CompleteStepController::renderCompleteView
+                        label: 'Order confirmation'
+                        translation_domain: checkout
+```
+
+
+## Configure shipping and payment payment_methods
+
+``` yaml 
+ibexa:
+    repositories:
+        <your_repository>:
+            checkout:
+                workflow: <your_workflow_identifier>
+                billing_address_format: <your_fieldtype_address_format_identifier> #coming from Corporate Account "billing" by default
+                shipping_address_format: <your_fieldtype_address_format_identifier> #coming from Corporate Account "shipping" by default 
+                customer_content_type: <your_ct_identifier_for_customer> #customer by default, it is used in registration and uses given shipping/billing addresses to pre-populate address forms in "Select Address" checkout step
+                shipping_methods:
+                    courier:
+                        name: "Courier"
+                        translation_domain: "checkout" #optional, all the shipping/payment methods use this one by default
+                    parcel_machine:
+                        name: "Parcel machine"
+                    self_pickup:
+                        name: "Self pickup"
+                    express_delivery:
+                        name: "Express delivery"
+                payment_methods:
+                    credit_card:
+                        name: "Credit card"
+                    paypal:
+                        name: "Paypal"
+                    money_transfer:
+                        name: "Money transfer"
+```
+
+## Shipping and billing address field configurations 
+
+The billing and shipping addresses used in checkout are related to their respective 
+values that exist in corporate account information. 
+You can configure these addresses by modifying the following code under `../ibexa/checkout/blob/main/src/bundle/DependencyInjection/Configuration/CheckoutParser.php`
```

Comment:
That is actually wrong - no code sitting in the vendor can or should be adjusted. For changing the default format for addresses one should create a format within Fieldtype Address configuration e.g. `foo` and reference it under `billing_address_format` / `shipping_address_format` checkout configuration keys, as stated above.

Reply:
Please review the updated description